src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my`
+        + ` ${person.sex === 'm' ? 'wife' : 'husband'}`}
+      </p>
+    ) : 'I am not married'}
```

Comment:
1. Split this logic and move part of it out of JSX.
2. `I am married` also should be in a `p` tag

Reply:
I'm not sure I correctly understood what you meant in the first one, but I kinda fixed it.